CONTRIBUTING.md

```diff
@@ -30,6 +30,48 @@ Recommended dev environment is
 6. Open http://127.0.0.1:8080/ in a browser to access Studio
 7. Register your development license in the OS environment as `RAVEN_License`
 
+## Building RavenDB artifacts
+
+Preconditions
+
+- .NET 6 SDKs
+- latest NodeJS LTS 
```

Comment:
maybe let's add Powershell here (https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux)

Reply:
link added for Linux users